openstack/openstack.c

```diff
@@ -76,7 +76,12 @@ void openstack_init(void) {
    
    //===== applications
    openapps_init();
-   
+

```

Comment:
This code is no necessary. When compiling, user already know printf is enabled. actually I think we can make it as default.